docs/CHANGELOG.asciidoc

```diff
@@ -0,0 +1,48 @@
+// Use these for links to issue and pulls. Note issues and pulls redirect one to
+// each other on Github, so don't worry too much on using the right prefix.
+// :issue: https://github.com/elastic/elasticsearch/issues/
+// :pull: https://github.com/elastic/elasticsearch/pull/
+
+= Elasticsearch Release Notes
+
+== Elasticsearch 7.0.0
+
+=== Breaking Changes
+
+=== Breaking Java Changes
```

Comment:
This section will always be empty in this repo.  But I'll leave it up to you whether you remove it or leave it for consistency with the elasticsearch repo.